packages/ses/test/global-lexicals-evaluate.test.js

```diff
@@ -0,0 +1,134 @@
+/* global Compartment */
+import test from 'tape';
+import '../src/main.js';
+
+test('endowments own properties are mentionable', t => {
+  t.plan(1);
+
+  const endowments = { hello: 'World!' };
+  const modules = {};
+  const compartment = new Compartment(endowments, modules);
+
+  const whom = compartment.evaluate('hello');
+  t.equal(whom, 'World!');
+});
+
+test('endowments own properties are enumerable', t => {
+  t.plan(1);
+
+  const endowments = { hello: 'World!' };
+  const modules = {};
+  const compartment = new Compartment(endowments, modules);
+
+  const keys = compartment.evaluate('Object.keys(globalThis)');
+  t.deepEqual(keys, ['hello']);
+});
+
+test('endowments prototypically inherited properties are not mentionable', t => {
+  t.plan(1);
+
+  const endowments = { __proto__: { hello: 'World!' } };
+  const modules = {};
+  const compartment = new Compartment(endowments, modules);
+
+  t.throws(() => compartment.evaluate('hello'), /hello is not defined/);
+});
+
+test('endowments prototypically inherited properties are not enumerable', t => {
+  t.plan(1);
+
+  const endowments = { __proto__: { hello: 'World!' } };
+  const modules = {};
+  const compartment = new Compartment(endowments, modules);
+
+  const keys = compartment.evaluate('Object.keys(globalThis)');
+  t.deepEqual(keys, []);
+});
+
+test('global lexicals are mentionable', t => {
+  t.plan(1);
+
+  const endowments = {};
+  const modules = {};
+  const globalLexicals = { hello: 'World!' };
+  const compartment = new Compartment(endowments, modules, { globalLexicals });
+
+  const whom = compartment.evaluate('hello');
+  t.equal(whom, 'World!');
+});
+
+test('global lexicals are not reachable from global object', t => {
+  t.plan(1);
+
+  const endowments = {};
+  const modules = {};
+  const globalLexicals = { hello: 'World!' };
+  const compartment = new Compartment(endowments, modules, { globalLexicals });
+
+  const keys = compartment.evaluate('Object.keys(globalThis)');
```

Comment:
This is testing enumerability, not reachability, right? Can we add a check that `globalThis.hello` is undefined?

Reply:
BTW I'm digging the `mentionable`/`reachable`/`enumerable` taxonomy.